app/views/wishlists_for_guests/_item.html.erb

```diff
@@ -0,0 +1,11 @@
+<li>
+  <p>
+    <strong>
+      <%= item.name %>
+    </strong>
+  </p>
+
+  <p>
+    <%= item.description %>
+  </p>
+</li>
```

Comment:
This is intentional since I think we need two separate ones.
The one in views/wishlists will have links for operations only user can do, e.g. delete an item, whereas this one will only allow for reserving an item.